test/integration/lib/study_search_service_test.rb

```diff
@@ -0,0 +1,126 @@
+require 'test_helper'
+
+class StudySearchServiceTest < ActiveSupport::TestCase
+
+  before(:all) do
+    @user = FactoryBot.create(:user, test_array: @@users_to_clean)
+    @metadata_study = FactoryBot.create(:detached_study,
+                                        name_prefix: 'Study Search Service Test',
+                                        public: true,
+                                        user: @user,
+                                        test_array: @@studies_to_clean)
+    @metadata_file = FactoryBot.create(:metadata_file,
+                                       name: 'metadata.txt',
+                                       study: @metadata_study,
+                                       annotation_input: [
+                                         {
+                                           name: 'cell_type',
+                                           type: 'group',
+                                           values: [
+                                             'B cell', 'amacrine cell', 'retinal cone cell'
+                                           ]
+                                         }
+                                       ])
+    @name_study = FactoryBot.create(:detached_study,
+                                    name_prefix: 'Cell Name Test',
+                                    public: false,
+                                    user: @user,
+                                    test_array: @@studies_to_clean)
+    @base_studies = Study.where(user: @user)
+    @filters = [
```

Comment:
can you add a filter to this list that doesn't match `cell` so that we confirm non-matching filters don't end up in the results?